### ISSUE TITLE
Add additional architectures to pylace build

### ISSUE THREAD

**Issue by schmidmt** [promised-ai/lace#51]
After polars is updated, we need to add x86 builds back to the automated builds in ci.

**Comment by Swandog** [comment]
Do you mean non-x86 builds?

**Comment by schmidmt** [comment]
You're correct. Though x86 jobs weren't failing due to explicit references to x86_64 methods.

**Comment by Swandog** [comment]
I'm not sure if there's a good solution for doing this in CI. GitHub doesn't have ARM runners available. And they also specifically suggest you DON'T use self-hosted runners for public projects, for security reasons.

We can still build the ARM packages, just not run them. GitHub says they will have ARM runners at some point; I don't know if this is worth working on until then

**Comment by Swandog** [comment]
This is what I'm basing my view on MacOS ARM runners on:

#https://github.com/github/roadmap/issues/528

**Comment by BaxterEaves** [comment]
Is this still an issue?

**Comment by Swandog** [comment]
I don't think the original issue is. We are still not running CI on ARM builds, but we probably won't be unless/until GitHub starts offering free ARM CI.
